### PR TITLE
Add enhancement 

### PR DESCRIPTION
**Pull request by lancehaha**

### CODE REVIEW

File: .github/workflows/jekyll-gh-pages.yml
Comment (charlesliyifeng) [inline .github/workflows/jekyll-gh-pages.yml:1]: What is this for?

File: .github/workflows/static.yml
Comment (charlesliyifeng) [inline .github/workflows/static.yml:1]: What is this for?

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1037,6 +1037,11 @@ Could not read data from file FILE_PATH due to inability to find or access the f
 This error message could be more specific and our team plans to update this to the following error message: <br>
 DorManagerPro could not access the file at FILE_PATH. This could be because the file is of the wrong type, it has a format incompatible with DorManagerPro or it contains invalid data. Please check that the file path leads to a JSON file with valid data and formatting.
 
+6. **Consistent behavior for `find`.** Currently, `find` are having inconsistent behavior on different fields which is counterintuitive. The problems are as following: <br>
```
Comment (chashaobao123) [inline docs/DeveloperGuide.md:1040]: maybe can rephrase to:

"Currently, `find` has some inconsistent behaviour depending on the field that it is being searched by, making it counterintuitive for users."

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1037,6 +1037,11 @@ Could not read data from file FILE_PATH due to inability to find or access the f
...
+The name search is case-insensitive, but tag search is not. We plan to modify `find t/` tag search to make it case-insensitive. <br>
+In the name search any contacts matching at least any one keyword will be returned (i.e. `OR` search). e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`. But phone number search requires the number to be exactly same with country code and region code. <br>
```
Comment (chashaobao123) [inline docs/DeveloperGuide.md:1042]: maybe can rephrase to:

"Searching by name filters contacts that match at least one keyword (i.e. `OR` search). E.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`. 
However, searching by phone number only filters contacts that match exactly to the optional country code, optional area code, and compulsory number. E.g. `98765432` will not return `+65 98765432`.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -1037,6 +1037,11 @@ Could not read data from file FILE_PATH due to inability to find or access the f
...
+We plan to make the phone search versatile and let user be able to use phone number without country code, or even only with country code to search<br>
```
Comment (chashaobao123) [inline docs/DeveloperGuide.md:1043]: maybe can rephrase to :

"We plan to modify `find p/` to allow searching by country code or area code only, and compulsory number only, so as to make this more versatile for users.

**Review by chashaobao123 — APPROVED** [review]

**Pull request merged**